http/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/http/service/AppServiceInterpSpec.scala

```diff
@@ -1907,6 +1907,75 @@ final class AppServiceInterpSpec extends AnyFlatSpec with LeonardoTestSuite with
     res.swap.toOption.get.getMessage shouldBe s"Persistent disk Gcp/${workspaceId}/disk is already formatted by CROMWELL"
   }
 
+  it should "V2 Azure - deleteAllApp, update all status appropriately, and queue multiple messages" in isolatedDbTest {
```

Comment:
Would it make sense to have a similar test with apps in Google as well since the V2 route is cloud context agnostic?